test/controllers/auth/MinimumAuthActionSpec.scala

```diff
@@ -88,30 +88,26 @@ class MinimumAuthActionSpec extends FreeSpec with MustMatchers with MockitoSugar
     }
   }
 
+  implicit class Ops[A](a: A) {
+    def ~[B](b: B): A ~ B = new ~(a, b)
+  }
+
   "A user with nino and no SA enrolment must" - {
     "create an authenticated request" in {
 
       val nino = Fixtures.fakeNino.nino
       val retrievalResult: Future[
         Option[String] ~ Enrolments ~ Option[Credentials] ~ ConfidenceLevel ~ Option[UserName] ~ LoginTimes ~ Option[
-          TrustedHelper]] =
+          TrustedHelper] ~ Option[String]] =
```

Comment:
Same issues as test/controllers/auth/AuthActionSpec.scala
Lots of repetition when building retrievals and the type can be put somewhere common